src/bci_build/package/python.py

```diff
@@ -159,3 +165,43 @@ def _get_python_kwargs(py3_ver: _PYTHON_VERSIONS, os_version: OsVersion):
     )
     for os_version in (OsVersion.SP7,)
 ]
+
+_CI_PYTHON = ("3.13", "313")
+
+
+BCI_CI_CONTAINERS = [
+    DevelopmentContainer(
+        name="bci-ci",
+        pretty_name="BCI-internal CI",
+        version=(python_ver := "%%python_version%%"),
```

Comment:
Not sure if we really need this? I mean will we ever pull an image by python version and not latest?